src/bun.js/web_worker.zig

```diff
@@ -622,6 +622,10 @@ pub fn exitAndDeinit(this: *WebWorker) noreturn {
         arena_.deinit();
     }
 
+    if (comptime Environment.isWindows) {
+        bun.windows.libuv.Loop.shutdown();
```

Comment:
Probably safer to move this to happen before the arena is deinitialized and before `deinit` is called instead of after. That way if any of the uv_close callbacks do end up being called they don't risk use-after-free.

Reply:
moved it above `deinit`